CONTRIBUTING.md

```diff
@@ -0,0 +1,229 @@
+<div align="Center">
+<h1>Contribution Guidelines for Avail's docs</h1>
+</div>
+
+- [How to Contribute to the Avail Documentation](#how-to-contribute-to-the-avail-documentation)
```

Comment:
We should re-format the heading hierarchy now that these sections are part of a new doc and not part of the heading hierarchy of the README.

Reply:
Done